### DISCUSSION TITLE
0.5.0

### DISCUSSION THREAD

**Discussion by mgmeyers** [mgmeyers/obsidian-kanban#150] [Releases]
- Thanks to @pjeby: Enable dragging cards and lists between boards

<hr /><em>This discussion was created from the release <a href='https://github.com/mgmeyers/obsidian-kanban/releases/tag/0.5.0'>0.5.0</a>.</em>

**Comment by troilus** [comment]
Nice！Hope it can make a copy by pressing ctrl while dragging.

**Comment by fuhadabdulla** [comment]
Thank You  Mg Meyer Kanban is one of the best plugin in the Obsidian .
